### PR TITLE
Region Migration: Set default it to IoTV2

### PR DESCRIPTION
**Pull request by Pengzna**
as title

currently, iotv2 will only fail in unstable ITs where iotv1 will fail as well.

for tests that will run per commit, iotv2 passed all

image
image

### CODE REVIEW

File: integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/pass/daily/datanodecrash/iotv2/IoTDBRegionMigrateDataNodeCrashForIoTV2IT.java [hunk 1]
```diff
@@ -97,14 +98,16 @@ public void destinationCrashDuringCreateLocalPeer() throws Exception {
...
   public void destinationCrashDuringAddPeerTransition() throws Exception {
-    failTest(
-        2,
-        2,
-        1,
-        3,
-        noKillPoints(),
-        buildSet(DataNodeKillPoints.DESTINATION_ADD_PEER_TRANSITION),
-        KillNode.DESTINATION_DATANODE);
+    if (isIoTV1) {
```
Comment (OneSizeFitsQuorum) [inline integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/pass/daily/datanodecrash/iotv2/IoTDBRegionMigrateDataNodeCrashForIoTV2IT.java:101]: iotv1?

File: integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ public class IoTDBRegionMigrateReliabilityITFramework {
   private static final String COUNT_TIMESERIES = "select count(*) from root.sg.**";
   private static final String REGION_MIGRATE_COMMAND_FORMAT = "migrate region %d from %d to %d";
   private static final String CONFIGURATION_FILE_NAME = "configuration.dat";
-  protected final boolean isIoTV1 = true;
+  protected boolean isIoTV1 = true;
```
Comment (OneSizeFitsQuorum) [inline integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java:93]: do we need this field anymore?

Comment (Pengzna) [reply integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/IoTDBRegionMigrateReliabilityITFramework.java]: fixed

File: integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/pass/daily/datanodecrash/iotv2/IoTDBRegionMigrateDataNodeCrashForIoTV2IT.java (hunk 1 above)
Comment (Pengzna) [reply integration-test/src/test/java/org/apache/iotdb/confignode/it/regionmigration/pass/daily/datanodecrash/iotv2/IoTDBRegionMigrateDataNodeCrashForIoTV2IT.java]: fixed

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**